app/assets/src/components/common/CoverageVizBottomSidebar/CoverageVizBottomSidebar.jsx

```diff
@@ -92,11 +93,11 @@ export default class CoverageVizBottomSidebar extends React.Component {
   };
 
   componentDidUpdate(prevProps, prevState) {
-    const { visible, params } = this.props;
+    const { params } = this.props;
     const { currentAccessionData } = this.state;
 
-    if (!prevProps.visible && visible) {
-      this.setCurrentAccession(params.accessionSummaries[2].id);
+    if (params.accessionSummaries !== prevProps.params.accessionSummaries) {
+      this.setCurrentAccession(get([2, "id"], params.accessionSummaries));
```

Comment:
This is not new... but a good pattern for this magic numbers to improve readability is to name it... I do not know what 2 means but you could define a constant like: `ACCESSION_THAT_MATTERS_INDEX=2`.
This would allows use to understand what we trying to load...

Reply:
This is temporary code for mock data. Will add a TODO.